packages/explat-client-react-helpers/src/index.tsx

```diff
@@ -68,19 +70,24 @@ export default function createExPlatClientReactHelpers(
 	};
 
 	const Experiment = ( {
-		children,
+		// As default is a keyword:
+		default: defaultExperience,
+		treatment,
+		loading,
```

Comment:
I wonder if it'd be better to avoid the use of the `default` keyword by making the parameter names more explicit and self-documenting? For example, we can call them `defaultExperience` / `treatmentExperience` / `loadingExperience` or `defaultNode` / `treatmentNode` / `loadingNode` or `defaultElement` / `treatmentElement` / `loadingElement`.